operatorconfig/moduleconfig/application-mobility/v0.3.0/certificate.yaml

```diff
@@ -18,4 +21,15 @@ spec:
   issuerRef:
     kind: Issuer
     name: <NAMESPACE>-selfsigned-issuer
-  secretName: cert-manager-webhook-ca
\ No newline at end of file
+  secretName: webhook-server-cert
+status:
```

Comment:
you can remove the status field in this file

Reply:
Fixed per commit 60a51c7e91da193c834c6f56c47b3f457f559f62